services/auth/token/helpers/token.js

```diff
@@ -0,0 +1,18 @@
+import to from 'await-to-js';
+import jwt from 'jsonwebtoken';
+import { throwError } from '@helsingborg-stad/npm-api-error-handling';
+
+const SECRET_KEY = 'secretKeey';
+
+export const signToken = jsonToSign => {
+  const token = jwt.sign(jsonToSign, SECRET_KEY);
+  return token;
+};
+
+export const verifyToken = async token =>
```

Comment:
verifyToken = async token =>...
verifyToken = async (token) =>...
 according to my linting rules